spec/scrabble_spec.js

```diff
@@ -1,9 +1,46 @@
 var Scrabble = require("../scrabble.js");
 
-describe('Hello World', function() {
+describe('errthang in scrabble.js', function() {
   var scrabble = new Scrabble();
 
   it('says hello world', function() {
     expect(scrabble.helloWorld()).toEqual('hello world!');
   });
+
+  it('knows it has points, Z=10', function() {
+    expect(scrabble._tilepoints['Z']).toEqual(10);
+  });
+
+  it('returns 5 for AEIOU', function() {
+    expect(scrabble.score('AEIOU')).toEqual(5);
+  });
+
+  it('is case insensitive', function() {
+    expect(scrabble.score('aeiou')).toEqual(5);
+  });
+
+  it('adds 50 for using 7 letters', function() {
+    expect(scrabble.score('AEIOUAA')).toEqual(57);
+  });
+
+  it('knows ZZZZZZZ is the highest word', function() {
+    expect(scrabble.highestScoreFrom(['AEIOU','ZZZZZZZ'])).toEqual('ZZZZZZZ');
+  });
+
```

Comment:
For more relabel tests, it's good idea to use longer arrays with edge cases words